locales/messages.de.json

```diff
@@ -51,23 +51,23 @@
   "LicenseExpressionUnknownException": "Unbekannter Lizenzausnahmebezeichner „{value}“. Bekannte Werte werden unter https://spdx.org/licenses/exceptions-index.html aufgeführt.",
   "LicenseExpressionUnknownLicense": "Unbekannter Lizenzbezeichner „{value}“. Bekannte Werte werden unter https://spdx.org/licenses/ aufgeführt.",
   "NoLocalizationForMessages": "Keine Lokalisierung für die folgenden Nachrichten:",
-  "ProcessorArchitectureMalformed": "Fehler beim Analysieren von %PROCESSOR_ARCHITECTURE% ({arch}) als eine gültige CPU-Architektur.",
+  "ProcessorArchitectureMalformed": "Fehler beim parsen von %PROCESSOR_ARCHITECTURE% ({arch}) als eine gültige CPU-Architektur.",
```

Comment:
according to the Duden `parsen` is a German word.

Reply:
But here used as a noun, so please Upper Case: `Parsen`

Reply:
(internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565237